src/transformers/tokenization_utils_base.py

```diff
@@ -1454,12 +1454,20 @@ def __init__(self, **kwargs):
         model_max_length = kwargs.pop("model_max_length", kwargs.pop("max_len", None))
         self.model_max_length = model_max_length if model_max_length is not None else VERY_LARGE_INTEGER
 
-        # Padding side is right by default and overridden in subclasses. If specified in the kwargs, it is changed.
+        # Padding and truncation side are right by default and overridden in subclasses. If specified in the kwargs, it
+        # is changed.
         self.padding_side = kwargs.pop("padding_side", self.padding_side)
         assert self.padding_side in [
             "right",
             "left",
         ], f"Padding side should be selected between 'right' and 'left', current value: {self.padding_side}"
```

Comment:
I propose to change the assert to an exception in [another PR](https://github.com/huggingface/transformers/pull/15454), so the 2 blocks of code will have the same format.

Reply:
Yup that seems consistent with the goal to remove assert in code.

Reply:
Change it here too, at least to maintain consistency with the following lines.